src/styles/_Grid.scss

```diff
@@ -11,6 +11,14 @@
   width: 100%;
 }
 
+.container-small {
```

Comment:
Given that we have different kinds of containers here we might want to refactor then in a better shape:

.container {
  &.small {}
  &.large {}
}